admin/public/styles/keystone/list.less

```diff
@@ -282,7 +282,7 @@
 			width: 16px;
 			height: 18px;
 			padding: 0px;
-			background: url('/keystone/images/icons/list-controls.png') top left no-repeat;
+			background: url('../images/icons/list-controls.png') top left no-repeat;
```

Comment:
Absolutely, just replace:
`background-image: url( "../images/icons/16/field-text.png" );`
with
`background-image: data-uri("../images/icons/16/field-text.png");`

The caveat being a heavier stylesheet